GooglePageTask/ClientApp/src/app/fetch-data/fetch-data.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Component({
+  selector: 'app-fetch-data',
+  templateUrl: './fetch-data.component.html'
+})
+export class FetchDataComponent {
+  public forecasts: WeatherForecast[];
+
+  constructor(http: HttpClient, @Inject('BASE_URL') baseUrl: string) {
+    http.get<WeatherForecast[]>(baseUrl + 'api/SampleData/WeatherForecasts').subscribe(result => {
+      this.forecasts = result;
+    }, error => console.error(error));
+  }
+}
+
+interface WeatherForecast {
+  dateFormatted: string;
+  temperatureC: number;
+  temperatureF: number;
+  summary: string;
+}
```

Comment:
Make sure interfaces and models should be in another files. Kindly go through given article on Teams